### PR TITLE
Fix module export type

### LINKED ISSUE #5: Module type is incorrect
The exported types imply that this module is an ES Module, but it's not. 

The correct way to export a CJS module is with `export =`, not `export default`

This error is silenced by `esModuleInterop: true` and should be removed. I suggest installing https://github.com/sindresorhus/tsconfig

### PR DESCRIPTION
**Pull request by fregante**
Fixes #5

### CODE REVIEW

**Comment by fregante** [comment]
Other than dubious coverage reductions, Travis reports [build success](https://travis-ci.org/github/SamVerschueren/map-age-cleaner/builds/732468742) (but GitHub conveniently doesn't show Travis anymore)

This is a semver breaking change.

**Comment by fregante** [comment]
Ping if you have time

**Comment by SamVerschueren** [comment]
Let me switch this to GitHub actions. Screw Travis...

File: source/test.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ test('auto removal on initial Map', async t => {
 
 	t.true(map.has('unicorn'));
 
-	await delay(600);
+	await delay(605);
```
Comment (SamVerschueren) [inline source/test.ts:26]: Any reason you changed this to 605? :)

**Review by SamVerschueren — COMMENTED** [review]
I've switched to GitHub actions, so I guess if you rebase it should be fine.

File: source/test.ts (hunk 1 above)
Comment (fregante) [reply source/test.ts:26]: It felt like 600 was “just enough” so I thought “a little more than just enough” would be more likely to trigger the issue, if it happened.

**Comment by fregante** [comment]
I’m AFK for the next few days. If you use Refined GitHub there’s a button/link to rebase it online.

File: source/test.ts (hunk 1 above)
Comment (SamVerschueren) [reply source/test.ts:26]: Aha I see, fair point!

**Pull request merged**